test/orchestratord/mzcompose.py

```diff
@@ -1650,6 +1700,7 @@ def measure_downtime() -> None:
                         preexec_fn=os.setpgrp,
                     )
                     connect_port_forward = False
+                time.sleep(1)
```

Comment:
This is not necessarily about this PR, but we might want a timeout on the select 1 query to prevent stalls if something goes wonky.

Reply:
If it stalls, the test should timeout or fail with an assertion at least.